tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/publicAsset/PublicAssetSetRequest.scala

```diff
@@ -211,4 +213,29 @@ case class PublicAssetUpdateResults(results: Seq[PublicAssetUpdateResult]) {
       case failure: PublicAssetUpdateFailure => Some(failure.id, failure.asSetError)
       case _ => None
     }).toMap
+}
+
+sealed trait PublicAssetDeletionResult
+case class PublicAssetDeletionSuccess(id: PublicAssetId) extends PublicAssetDeletionResult
+case class PublicAssetDeletionFailure(id: UnparsedPublicAssetId, exception: Throwable) extends PublicAssetDeletionResult {
+  def asSetError: SetError = exception match {
+    case e: PublicAssetException => SetError.invalidArguments(SetErrorDescription(e.getMessage))
```

Comment:
Log meeee ?

Reply:
Log at line 225 is not enough?
We should log all errors, not only serverFail?

Reply:
We shall IMO log INFO on syntax errors.